WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/logic/event/EventPersister.java

```diff
@@ -0,0 +1,32 @@
+package nl.tudelft.watchdog.intellij.logic.event;
+
+import nl.tudelft.watchdog.core.logic.event.EventPersisterBase;
+import nl.tudelft.watchdog.intellij.WatchDogStartUp;
+
+import java.io.File;
+
+/**
+ * {@inheritDoc} IntelliJ's version of EventPersister which handles ClassLoaders.
+ */
+public class EventPersister extends EventPersisterBase {
```

Comment:
Couldn't this be even more generic as the same code should be needed in the IntervalPersister?